### PR TITLE
Added footer banner and redesigned the footer

### PR DESCRIPTION
**Pull request by mcbobby7**
Here i added the green banner and redesigned the footer adding extra links and pushing the copyright to be under and in its own segment. ( i did not use selectors, i was already done before i go to know about the selectors, i can still refracture  required )

### CODE REVIEW

File: src/components/app/components/banner.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
+import { Button } from 'antd';
+
+class Banner extends Component {
```
Comment (EngrEric) [inline src/components/app/components/banner.js]: @mcbobby7 remove all inline styling

Comment (mcbobby7) [reply src/components/app/components/banner.js]: Ok

File: src/components/app/components/banner.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                                tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+                                
+                                </p></h5>
+                            </div>
+                        </div>
+                        <Button>Learn More</Button>
+                    </div>
+        );
+    }
+}
```
Comment (Way2nnadi) [inline src/components/app/components/banner.js:29]: @EngrEric can you show @mcbobby7 how to store all the information in the an array of objects and how to map over that array to display the content

File: src/components/app/components/footer2.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import './styles.css';
+
+
+class Footer2 extends Component {
+    render() {
+        return (
+                <div className="footer2">
+                    <h4>All rights reserved, Copyright © 2019 Agroeasy.</h4>
+                </div>
+        );
```
Comment (Way2nnadi) [inline src/components/app/components/footer2.js:13]: store all strings in the `constants.js` file

File: src/components/app/constants.js [hunk 4]
```diff
@@ -64,10 +64,18 @@ export const FOOTER_DATA = [{
...
     key: 'support',
-}, {
-    items: [{ title: 'All rights reserved, Copyright © 2018 Agroeasy.' }],
+},
+ {
+    header: 'AgroEasy',
+    items: [{ title: 'Privacy Policy' },
+            { title: 'Terms And Conditions' },
+            { title: 'Report Abuse' }
+           ],
```
Comment (Way2nnadi) [inline src/components/app/constants.js]: formatting is wrong
```
items: [
    { title: 'Privacy Policy' },
    { title: 'Terms And Conditions' },
    { title: 'Report Abuse' },
],
```

**Comment by Way2nnadi** [comment]
@mcbobby7 your build is failing, run `npm run lint` to see all the issues that you need to fix

File: src/components/app/constants.js (hunk 4 above)
Comment (mcbobby7) [reply src/components/app/constants.js]: OK will do that

File: src/components/app/components/banner.js [hunk 5]
```diff
@@ -1,58 +1,29 @@
...
-                            </div>
-                        </div>
-                        <Button>Learn More</Button>
-                    </div>
+class Banner extends Component { 
+    render() {
+        return (
+            <div className={BANNER}>
+                <h2>{BANNER_HEADING}</h2>
+                <div className={FLEXBOX}>{banners}</div>
```
Comment (enyeInc) [inline src/components/app/components/banner.js]: Do not name this className flexbox. Name is `banners` or `banner-container` given that it contains the banners`

**Review by enyeInc — APPROVED** [review]
Looks good, just need a few minor fix and also remember to bump the version

File: src/components/app/components/Footer.js [hunk 6]
```diff
@@ -18,7 +18,7 @@ const footers = FOOTER_DATA.map(footer => (
             ))
         }
     </div>
-));
+));
```
Comment (enyeInc) [inline src/components/app/components/Footer.js:21]: why is this here?

Comment (mcbobby7) [reply src/components/app/components/Footer.js:21]: @enyeinc sorry I don't understand

Comment (enyeInc) [reply src/components/app/components/Footer.js:21]: delete that added space

Comment (mcbobby7) [reply src/components/app/components/Footer.js:21]: Ok

File: src/components/app/constants.js [hunk 7]
```diff
@@ -78,6 +91,47 @@ export const FOOTER_CLASSNAMES = {
...
+        key: 'price',
+    }, 
+    {
+        icon: 'fa fa-trash',
+        header: 'Reduce Food Wastage',
+        body: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmodtempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam',
+        key: 'food',
+    }, 
+    {
+        icon: 'fa fa-users',
```
Comment (enyeInc) [inline src/components/app/constants.js]: @mcbobby7 where are these icons coming from? are they AntD icons?

Comment (mcbobby7) [reply src/components/app/constants.js]: No I used font awesome

Comment (enyeInc) [reply src/components/app/constants.js]: Use AntD icons

Comment (mcbobby7) [reply src/components/app/constants.js]: Ok

File: src/components/app/components/App.js [hunk 8]
```diff
@@ -4,16 +4,18 @@ import { connect } from 'react-redux';
...
 import Footer from './Footer';
+import Banner from './banner';
 import Navbar from './Navbar';
 import { getLoginStatus, getUserData } from '../selectors';
 import { SIGNIN_SUCCESS } from '../constants';
 import { resetStatusStatus } from '../actions';
 
 const { ContactUs } = contactus.components;
-const { Content } = Layout;
+const { Content } = Layout;
```
Comment (enyeInc) [inline src/components/app/components/App.js]: delete extra space

File: src/components/app/constants.js [hunk 9]
```diff
@@ -64,12 +66,23 @@ export const FOOTER_DATA = [{
...
+    header: 'AgroEasy',
+    items: [
+        { title: 'Privacy Policy' },
+        { title: 'Terms And Conditions' },
+        { title: 'Report Abuse' },
+    ],
     key: 'rights',
-}];
+},
+];
```
Comment (enyeInc) [inline src/components/app/constants.js]: revert this change `}];` is correct

File: src/components/app/constants.js [hunk 10]
```diff
@@ -43,7 +45,7 @@ export const USER_AVATAR = {
     SIGN_OUT: "Sign Out",
     USER_DROP_DOWN: "user-avatar-drop-down",
     USER_PROFILE: "User Profile",
-};
+};
```
Comment (enyeInc) [inline src/components/app/constants.js:48]: delete this space

File: src/components/app/constants.js [hunk 11]
```diff
@@ -13,7 +15,7 @@ export const LOGO = {
     SHAPE: "square",
     SIZE: "large",
     SOURCE: "/AgroeasyLogo.png",
-};
+};
```
Comment (enyeInc) [inline src/components/app/constants.js]: delete this space

File: src/components/app/components/footer2.js [hunk 12]
```diff
@@ -0,0 +1,16 @@
+import React, { Component } from "react";
+import './styles.css';
+import { FOOTER_STRING } from '../constants';
+
+class Footer2 extends Component {
+    render() {
+        return (
+            <div className="footer2">
```
Comment (enyeInc) [inline src/components/app/components/footer2.js]: add className string to `constants.js` file

File: src/index.html [hunk 13]
```diff
@@ -14,3 +13,4 @@
 </body>
 
 </html>
+
```
Comment (enyeInc) [inline src/index.html:13]: delete extra line
